mars/services/task/analyzer/analyzer.py

```diff
@@ -303,6 +303,12 @@ def gen_subtask_graph(self) -> SubtaskGraph:
             "Assigned %s start chunks for task %s", len(start_ops), self._task.task_id
         )
 
+        # assign expect workers for successors
+        for chunk in self._chunk_graph.topological_iter():
```

Comment:
No need to use `topological_iter`, `for chunk in self._chunk_graph` is more efficient。

Reply:
I've fixed this.